query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverterTest.java

```diff
@@ -46,27 +48,21 @@
   private static final String TENANT_COLUMN_NAME = "tenant_id";
 
   private static final String TEST_REQUEST_HANDLER_CONFIG_FILE = "request_handler.conf";
+  private static final String TEST_SERVICE_REQUEST_HANDLER_CONFIG_FILE =
+      "service_request_handler.conf";
 
-  private static ViewDefinition viewDefinition;
+  private ExecutionContext mockingExecutionContext;
   private Connection connection;
 
-  @BeforeAll
-  public static void setUp() {
-    Config fileConfig =
-        ConfigFactory.parseURL(
-            requireNonNull(
-                QueryRequestToPinotSQLConverterTest.class
-                    .getClassLoader()
-                    .getResource(TEST_REQUEST_HANDLER_CONFIG_FILE)));
-    viewDefinition =
-        ViewDefinition.parse(
-            fileConfig.getConfig("requestHandlerInfo.viewDefinition"), TENANT_COLUMN_NAME);
-  }
-
   @BeforeEach
   public void setup() {
-    connection = Mockito.mock(Connection.class);
+    connection = mock(Connection.class);
     Mockito.when(connection.prepareStatement(any(Request.class))).thenCallRealMethod();
+    mockingExecutionContext = mock(ExecutionContext.class);
+    when(mockingExecutionContext.getTenantId()).thenReturn("__default");
+    when(this.mockingExecutionContext.getTimeSeriesPeriod()).thenReturn(Optional.empty());
```

Comment:
this mock is not relevant to every test and would fail in strict mode. please move it such that we only mock things that are relevant to the test.

For future tests (beyond the scope of this PR), make sure to always run in strict mode. The easiest way to do that is probably use the junit extension -
```java
@ExtendWith(MockitoExtension.class)
```